src/Containers/App/index.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component, Suspense, lazy } from 'react';
+
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import './style.css';
+import BlobField from 'src/BlobField';
+import config from '../../config';
+
+export default class App extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = { width: 0, height: 0 };
+    this.updateWindowDimensions = this.updateWindowDimensions.bind(this);
+  }
+
+  componentDidMount() {
+    this.updateWindowDimensions();
+    window.addEventListener('resize', this.updateWindowDimensions);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.updateWindowDimensions);
+  }
+
+  updateWindowDimensions() {
+    this.setState({ width: window.innerWidth, height: window.innerHeight });
+  }
+
+  render() {
+    return (
+      <Router>
+        <div className="app">
+          <Suspense fallback={<div />}>
+            <Switch>
+              {
+                config.artists.map(({slug, component}) => {
+                  const Comp = lazy(() => import(`../../work/${component}`));
```

Comment:
Never seen this before. Rad!

Reply:
Yeah it is great. We had a single chunk at 500k, not too bad really. But this gives a big one at 155, then a smaller chunk for each artist route that gets loaded as-needed.